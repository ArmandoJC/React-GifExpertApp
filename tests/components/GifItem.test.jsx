import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";


describe("Pruebas en GifItem", () => {
  const title = 'One Punch man';
  const url = 'https://media2.giphy.com/media/d2PqO9s7M9CNO/giphy.gif?cid=596cc03ceyfrxyhhbowg1ni09pypku0cvaaiiru0blztu1no&rid=giphy.gif&ct=g';


  test("DeberÃ­a hacer match con el snapshot", () => {
    // render(<GifItem title={ title } url={ url } />);
    const { container } = render(<GifItem title={ title } url={ url } />);
    expect(container).toMatchSnapshot();
  
  });
});
