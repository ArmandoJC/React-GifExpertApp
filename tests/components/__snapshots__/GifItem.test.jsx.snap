// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifItem Deber√≠a hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="One Punch man"
      src="https://media2.giphy.com/media/d2PqO9s7M9CNO/giphy.gif?cid=596cc03ceyfrxyhhbowg1ni09pypku0cvaaiiru0blztu1no&rid=giphy.gif&ct=g"
    />
    <p>
      One Punch man
    </p>
  </div>
</div>
`;
